microsoft/testsuites/core/kvp.py

```diff
@@ -31,7 +31,8 @@
 )
 class Kvp(TestSuite):
     # lrwx------ 1 root root 64 Oct 18 19:05 9 -> /dev/vmbus/hv_kvp
-    _hv_kvp_pattern = re.compile(r".* /dev/vmbus/(hv_kvp)$", re.M)
+    # lrwx------. 1 root root 64 Dec 18 07:38 4 -> /dev/vmbus/hv_kvp
+    _hv_kvp_pattern = re.compile(r".* /dev/vmbus/(hv_kvp)\r?$", re.M)
```

Comment:
is a new line character in the second sample?

Reply:
No, I thought its format is different, but actually the reason is the \r at the end. I will remove the second sample.